### ISSUE TITLE
[Check] JavaScript: SQLi with string formatting utilities

### ISSUE THREAD

**Issue by inkz** [returntocorp/sgrep-rules#598]
**Check Description**

cover string formatting utils like `util.format(...)` and `sf(...)`

```javascript
    var query = 'INSERT INTO something(id, date)  VALUES (%s, %s)';
    connection.query(util.format(query, req.id, req.date), function(err) {
      if (err) return cb("error");
      cb("success");
    });
```

```javascript
    var query = 'INSERT INTO something(id, date)  VALUES ('{0}','{1}')';
    connection.query(sf(query, req.id, req.date), function(err) {
      if (err) return cb("error");
      cb("success");
    });
```

- [x] This ticket has links, references, or examples.
- [ ] Your check has true positive and true negative test cases.
- [ ] Your check has been labeled with true positive and false positive findings on at least 10 repositories, and the link is pasted in the ticket. If there are no findings, paste the triager link anyway.
- [ ] Your check PR has been reviewed and merged.
Now, close the ticket!

**Comment by inkz** [comment]
double checked: the scenario is already covered by sqli rule https://github.com/returntocorp/semgrep-rules/blob/develop/contrib/nodejsscan/sql_injection.yaml
